Tests/GRPCCoreTests/Call/Client/Internal/ClientRPCExecutorTests.swift

```diff
@@ -150,12 +150,12 @@ final class ClientRPCExecutorTests: XCTestCase {
       server: .failTest
     )
 
-    try await tester.unary(
-      request: ClientRequest.Single(message: [1, 2, 3], metadata: ["foo": "bar"])
-    ) { response in
-      XCTAssertThrowsRPCError(try response.message) {
-        XCTAssertEqual($0.code, .aborted)
-      }
+    await XCTAssertThrowsRPCErrorAsync({
+      try await tester.unary(
+        request: ClientRequest.Single(message: [1, 2, 3], metadata: ["foo": "bar"])
+      ) { _ in }
+    }) {
+      XCTAssertEqual($0.code, .aborted)
     }
```

Comment:
nit: use multiple trailing closure syntax (might be `errorHandler` instead of `onError`, don't remember off the top of my head)

```suggestion
    await XCTAssertThrowsRPCErrorAsync {
      try await tester.unary(
        request: ClientRequest.Single(message: [1, 2, 3], metadata: ["foo": "bar"])
      ) { _ in }
    } onError: {
      XCTAssertEqual($0.code, .aborted)
    }
```